public/index.html

```diff
@@ -3,24 +3,81 @@
   <head>
     <meta charset="utf-8">
     <title>Hoodie</title>
-  </head>
-  <body>
-    <h1>Hello from Hoodie!</h1>
+    <style>
+    body{
+background-color:#FCFCFC;
+animation-name: graytocolorimg;
+animation-duration: 4s;
+}
+h1{
+  font-family:fantasy,sans-serif;
```

Comment:
I’d say let’s focus more on content for now and keep the styling simple until we have a CI?